app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,369 @@
+            android:contentDescription="@string/evans"
+            android:src="@drawable/evans"
+            app:layout_constraintEnd_toStartOf="@+id/ruffalo_iv"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/downey_jr_iv"
+            app:layout_constraintTop_toBottomOf="@id/cast_tv" />
+
+        <TextView
+            android:id="@+id/evans_tv"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x1"
+            android:text="@string/evans"
+            android:textColor="@color/white"
+            android:textSize="@dimen/common_text_size"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/evans_barrier_start"
+            app:layout_constraintEnd_toStartOf="@id/evans_barrier_end"
+            app:layout_constraintTop_toBottomOf="@id/evans_iv" />
+
+        <ImageView
+            android:id="@+id/ruffalo_iv"
+            android:scaleType="fitCenter"
+            android:adjustViewBounds="true"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x2"
+            android:contentDescription="@string/ruffalo"
+            android:src="@drawable/ruffalo"
+            app:layout_constraintEnd_toStartOf="@+id/hemsworth_iv"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/evans_iv"
+            app:layout_constraintTop_toBottomOf="@id/cast_tv" />
+
+        <TextView
+            android:id="@+id/ruffalo_tv"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x1"
+            android:text="@string/ruffalo"
+            android:textColor="@color/white"
+            android:textSize="@dimen/common_text_size"
+            android:textStyle="bold"
+            app:layout_constraintStart_toEndOf="@id/ruffalo_barrier_start"
+            app:layout_constraintEnd_toStartOf="@id/ruffalo_barrier_end"
+            app:layout_constraintTop_toBottomOf="@id/ruffalo_iv" />
+
+        <ImageView
+            android:id="@+id/hemsworth_iv"
+            android:scaleType="fitCenter"
+            android:adjustViewBounds="true"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_x2"
+            android:contentDescription="@string/hemsworth"
+            android:src="@drawable/hemsworth"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/ruffalo_iv"
+            app:layout_constraintTop_toBottomOf="@id/cast_tv" />
+
+        <TextView
+            android:id="@+id/hemsworth_tv"
```

Comment:
What will happen if we decide to show a different actor in the specific place? Should we abstract content from the view?

Reply:
Agreed, hadn't thought about that. Changed to 1st_actor, 2nd_actor, etc